### PR TITLE
[Merged by Bors] - Adds a "Question" link to the new issue selection

### LINKED ISSUE #5150: Add issue templates for "ask a question" that redirects to Discussions
## What problem does this solve or what need does it fill?

Users sometimes file unhelpful issues when asking questions about how to use Bevy.

These have to be manually converted to Discussions, which is a more hostile process than I would like.

## What solution would you like?

Create an "issue template" that links to the discussion instead.

Seen on https://github.com/rust-lang/rust/issues/new/choose

image

## Additional context

Appears to be configured using [this config.yml file](https://github.com/rust-lang/rust/blob/master/.github/ISSUE_TEMPLATE/config.yml).

### PR DESCRIPTION
**Pull request by themasch**
This gives users a hint to use the GitHub "Discussions" for questions about bevy, instead of filing an issue.

We should provide a link to the Discussion board in the list of "new Issue" options. This hopefully allows users to better find this option and reduces the number of question-issues.

- fixes #5150

## Solution

- add a small config.yml that configures the link

Looks like this (currently live on my local fork https://github.com/themasch/bevy/issues/new/choose):
grafik

---

## Open questsions
 - I am unsure about the wording.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: Question
+    url: https://github.com/bevyengine/bevy/discussions
+    about: Please ask and answer questions about Bevy in the "Discussion" section.
```
Comment (alice-i-cecile) [inline .github/ISSUE_TEMPLATE/config.yml:4]:
```suggestion
    about: Questions about how to use or contribute to Bevy belong in Github Discussions
```

**Comment by alice-i-cecile** [comment]
Thanks :) I've left a wording suggestion for you.

**Review by james7132 — APPROVED** [review]

**Comment by james7132** [comment]
Missed Alice not giving an explicit approval yet, whoops.

**Review by alice-i-cecile — APPROVED** [review]

**Comment by alice-i-cecile** [comment]
Controversial label added due to high user-visible impact :)

**Comment by Nilirad** [comment]
Couldn't it directly link to the new discussion page, possibly with Q&A selected?

**Comment by alice-i-cecile** [comment]
Clever idea. I'm split on that; the current design encourages searching first.

**Comment by Nilirad** [comment]
So I think the comment beside the button should guide people searching for an answer first.

**Review by IceSentry — APPROVED** [review]

**Comment by themasch** [comment]
So I could change the link to `https://github.com/bevyengine/bevy/discussions/categories/q-a` to pre-select the Q&A category, and/or change the comment to something like 
> Questions about how to use or contribute to Bevy belong in Github Discussions. You can use the search to check if someone already answered that question!

**Review by Nilirad — APPROVED** [review]
Looks good on mobile! I just have to scroll a little bit to read the description, but for the rest it's all visible.

I don't know if it's possible, but could changing the button color make it stand out even more? That gray looks a little dull.

**Comment by themasch** [comment]
> I don't know if it's possible, but could changing the button color make it stand out even more? That gray looks a little dull.

From what I understand right now, thats not possible. For those links you basically got three properties: `name`, `url` and `about`. 
The offical github docs about that are a bit terse on the details, but that might be because there are no details. A code search for what other people are doing did not turn up anything either.  The schema file that ships with the IDEA community edition also only knows these three properties.

**Review by cart — APPROVED** [review]
This is a really cool way to improve both Bevy maintainers' lives and new users'. Much appreciated!

**Pull request closed without merging**